clients/spark/core/src/main/scala/io/treeverse/clients/BulkRemoverFactory.scala

```diff
@@ -99,32 +93,24 @@ object BulkRemoverFactory {
       }
     }
 
-    private def getS3Client(
-        hc: Configuration,
-        bucket: String,
-        region: String,
-        numRetries: Int
-    ): AmazonS3 =
-      io.treeverse.clients.conditional.S3ClientBuilder.build(hc, bucket, region, numRetries)
-
     override def getMaxBulkSize(): Int = {
       S3MaxBulkSize
     }
   }
 
   private class AzureBlobBulkRemover(hc: Configuration, storageNamespace: String)
       extends BulkRemover {
-    val EmptyString = ""
-    val uri = new URI(storageNamespace)
-    val storageAccountUrl = StorageUtils.AzureBlob.uriToStorageAccountUrl(uri)
-    val storageAccountName = StorageUtils.AzureBlob.uriToStorageAccountName(uri)
+    private val EmptyString = ""
```

Comment:
I don't think we ever need this: Scala checks string equality by value so you can just `s != ""`.  (Also the naming is _really_ weird, it looks like a class).

Reply:
True. This is not a new var, the only thing that changed is the `private` scoping.
I'll take a llok.